python/ray/rllib/utils/seed.py

```diff
@@ -0,0 +1,13 @@
+from __future__ import absolute_import
```

Comment:
we should support seeding (this utility was incredibly useful during debugging). If so we also need to support environment seeding, which is not covered in this utility.